### PR TITLE
V0.12.0 rc

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR includes updates to documentation, release notes, build scripts, etc. for the upcoming RAJA release.
- It will continue to be updated up until the release, so no need to submit a review now. Making it a PR so folks can start looking it over if the want.

### CODE REVIEW

File: docs/sphinx/user_guide/getting_started.rst [hunk 1]
```diff
@@ -56,35 +54,31 @@ After running the clone command, a copy of the RAJA repository will reside in
...
-.. _getting_started_building-label:
+Building and installing RAJA can be very easy or more complicated, depending
+on which features you want to use and how easy it is to use your system.
 
 --------------
 Building RAJA
 --------------
 
-RAJA uses CMake to configure a build. A basic configuration looks like::
+RAJA uses CMake to configure a build. A "bare boes" configuration looks like::
```
Comment (willkill07) [inline docs/sphinx/user_guide/getting_started.rst:81]: Bare bones

Comment (rhornung67) [reply docs/sphinx/user_guide/getting_started.rst]: Thanks. Fixed.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -18,7 +120,7 @@ This release contains new features, several notable changes, and some bug fixes.
 Notable changes include:
 
   * New features:
-      * HIP compiler back-end added to support AMD GPUs. Usage is essentially
+      * Hip compiler back-end added to support AMD GPUs. Usage is essentially
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md:123]: I think this should be "HIP"

Comment (rhornung67) [reply RELEASE_NOTES.md]: Fixed. Thanks.

File: scripts/lc-builds/blueos_nvcc11_xl-2020.08.24.sh [hunk 3]
```diff
@@ -7,7 +7,12 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ###############################################################################
 
-BUILD_SUFFIX=lc_blueos-nvcc10-xl_2019.06.12
+#
+## NOTE: After building code, you need to load the cuda 11 module to run
```
Comment (davidbeckingsale) [inline scripts/lc-builds/blueos_nvcc11_xl-2020.08.24.sh:11]: Do you think it would be a good idea to echo this to the screen?

Comment (rhornung67) [reply scripts/lc-builds/blueos_nvcc11_xl-2020.08.24.sh:11]: We could do that. I didn't think about it. At this point, I just want to get this release out. I will merge this later today and do the release.

**Pull request closed without merging**